func/internal/podevaluator.go

```diff
@@ -57,6 +58,9 @@ const (
 	fieldManagerName          = "krm-function-runner"
 	functionContainerName     = "function"
 	defaultManagerNamespace   = "porch-system"
+	defaultRegistry           = "gcr.io/kpt-fn/"
+	// perhaps should try and get the name of the dockerconfig secret given by user and match this secret name to that to avoid hard coded value?
```

Comment:
Do we need this comment?

Reply:
no this will be removed with a new commit.

Reply:
amended in latest commit.